### ISSUE TITLE
Project grading

### ISSUE THREAD

**Issue by MatthiasYeoEeLiang** [nus-cs2030/2122-s1#352]
Even if I fail test cases, will my implementation still be considered in the grading rubrics?

**Comment by cadencjk** [comment]
#149 Partial marks will be given for completing levels in the project
#310 Apparently we are graded on our design as well, so I think your implementation may be considered in the grading (not sure).

**Comment by aaronthamsc98** [comment]
My TA also mentioned about grading based on the number of test cases that you passed, so try to get as many!

**Comment by kxshxsh** [comment]
yeah my TA said that grading will be done on codecrunch, according to its grader. So I'm guessing that implies heavy reliance on the number of testcases passed. All the best!
